heroes.js

```diff
@@ -20,7 +20,7 @@ function createHeroBlock(hero) {
 
 function setupHeroReveal() {
   for (var i = 0; i < heroes.length; i++) {
-    var hero = heroes[i]
+    const hero = heroes[i]
```

Comment:
Good approach, var is not block scoped but const is. By using const you've created hero in the scope of the for loop and stopped it from bleeding out. The other option would have been to switch to a `forEach` since the creation of the inner function would also have creating a new scope.